src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -17,15 +17,15 @@
 package com.google.cloud.tools.appengine.api.devserver;
 
 import java.io.File;
+import java.nio.file.Path;
 import java.util.List;
 
 /**
  * Plain Java bean implementation of {@link RunConfiguration}.
  */
 public class DefaultRunConfiguration implements RunConfiguration {
 
-  private List<File> appYamls;
-  private List<File> services;
+  private List<Path> services;
```

Comment:
How did it mess it up?
If this is being annotated on the Gradle side for the value to be automatically initialized, I'm wondering if we could add another field to Gradle's configuration and then translate those Files into Paths at some point in the run workflow?